thoth/metrics_exporter/jobs/adviser.py

```diff
@@ -18,20 +18,33 @@
 """Adviser metrics."""
 
 import logging
+import os
+from datetime import datetime
 
 from thoth.storages import GraphDatabase
 from thoth.storages.graph.enums import SoftwareStackTypeEnum
 import thoth.metrics_exporter.metrics as metrics
+from prometheus_api_client import PrometheusConnect
 
 from .base import register_metric_job
 from .base import MetricsBase
+from .common import get_workflow_duration
 
 _LOGGER = logging.getLogger(__name__)
 
 
 class AdviserMetrics(MetricsBase):
     """Class to evaluate Metrics for Adviser."""
 
+    _URL = "https://prometheus-dh-prod-monitoring.cloud.datahub.psi.redhat.com"
```

Comment:
Can this be parametrized using env variables so that we can transparently support other deployments, such as MOC?

Reply:
should we add a new env variable to thoth deployment?
```             
- name: PROMETHEUS_URL
        valueFrom:
        configMapKeyRef:
        name: thoth
        key: metrics-prometheus-url
```

Reply:
IMHO that would be the best solution :+1: